src/util.cpp

```diff
@@ -936,3 +927,10 @@ int64_t GetStartupTime()
 {
     return nStartupTime;
 }
+
+fs::path GetAbsolutePathForArg(const fs::path& path, bool net_specific)
+{
+    if (!path.is_complete())
```

Comment:
Could eliminate this check, but if you will keep it [developer notes](https://github.com/bitcoin/bitcoin/blob/master/doc/developer-notes.md) suggest adding braces or putting if body on same line as condition.

Reply:
Taking your advice below, but the reminder about convention is appreciated.